src/mono/mono/tests/verifier/make_il_overflow_test.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/sh
 
 SED="sed"
-if [ `which gsed 2> /dev/null` ] ; then 
+if [ `which gsed 2> /dev/null` ] ; then
```

Comment:
```suggestion
if [ `which gsed 2> /dev/null` ]; then
```

Reply:
Changed all occurrences of `] ;` with `];`.